guides/common/modules/proc_configuring-pull-transport.adoc

```diff
@@ -0,0 +1,31 @@
+[id="configuring-pull-provider_{context}"]
+= Configuring Remote Execution for Pull Client
+
+By default, Remote Execution uses SSH as the transport mechanism for the Script provider.
```

Comment:
what do you mean by "script provider"?

Reply:
Remote execution has several providers - Script (formerly SSH) and Ansible being the most common. The provider decides which job template is used.

Reply:
Just checked on Foreman 3.3 that "Script" is new. So capitalizing it is IMHO very much correct. 👍